packages/angular_devkit/build_angular/src/builders/application/schema.json

```
@@ -452,6 +452,26 @@
         }
       ]
     },
+    "ssr": {
+      "description": "Server side render (SSR) pages of your application during runtime.",
+      "default": false,
+      "oneOf": [
+        {
+          "type": "boolean",
+          "description": "Enable the server bundles to be written to disk."
+        },
+        {
+          "type": "object",
+          "properties": {
+            "entry": {
+              "type": "string",
+              "description": "The server entry-point that when executed will spawn the server."
```

```suggestion
              "description": "The server entry-point that when executed will spawn the web server."
```
Just to make it a little more explicit as to the options purpose